### DISCUSSION TITLE
Range Error while restoring backup?

### DISCUSSION THREAD

**Discussion by Heidieker** [TriliumNext/Trilium#4473] [Q&A]
What does it mean?
I use it in a docker on my synology. The whole RAM of 18GB is used by trilium. Because of this reinstalled Trilium an tried a backup.

`
023/11/24 13:34:40 | stderr | RangeError: Maximum call stack size exceeded
-- | -- | --
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | ^
2023/11/24 13:34:40 | stderr | }
2023/11/24 13:34:40 | stderr | /usr/src/app/src/services/notes.js:894
2023/11/24 13:34:40 | stderr | Exception in PromiseRejectCallback:
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | RangeError: Maximum call stack size exceeded
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | ^
2023/11/24 13:34:40 | stderr | }
2023/11/24 13:34:40 | stderr | /usr/src/app/src/services/notes.js:894
2023/11/24 13:34:40 | stderr | Exception in PromiseRejectCallback:
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | RangeError: Maximum call stack size exceeded
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | ^
2023/11/24 13:34:40 | stderr | }
2023/11/24 13:34:40 | stderr | /usr/src/app/src/services/notes.js:894
2023/11/24 13:34:40 | stderr | Exception in PromiseRejectCallback:
2023/11/24 13:34:40 | stdout | Saving image 'inline image' as attachment into note 'LmgibsjtmmzB'
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | RangeError: Maximum call stack size exceeded
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | ^
2023/11/24 13:34:40 | stderr | }
2023/11/24 13:34:40 | stderr | /usr/src/app/src/services/notes.js:894
2023/11/24 13:34:40 | stderr | Exception in PromiseRejectCallback:
2023/11/24 13:34:40 | stdout | Saving image 'inline image' as attachment into note 'LmgibsjtmmzB'
2023/11/24 13:34:40 | stdout | Saving image 'inline image' as attachment into note 'LmgibsjtmmzB'
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | RangeError: Maximum call stack size exceeded
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | ^
2023/11/24 13:34:40 | stderr | }
2023/11/24 13:34:40 | stderr | /usr/src/app/src/services/notes.js:894
2023/11/24 13:34:40 | stderr | Exception in PromiseRejectCallback:
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | RangeError: Maximum call stack size exceeded
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | ^
2023/11/24 13:34:40 | stderr | }
2023/11/24 13:34:40 | stderr | /usr/src/app/src/services/notes.js:894
2023/11/24 13:34:40 | stderr | Exception in PromiseRejectCallback:
2023/11/24 13:34:40 | stdout | Saving image 'inline image' as attachment into note 'LmgibsjtmmzB'
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | RangeError: Maximum call stack size exceeded
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | ^
2023/11/24 13:34:40 | stderr | }
2023/11/24 13:34:40 | stderr | /usr/src/app/src/services/notes.js:894
2023/11/24 13:34:40 | stderr | Exception in PromiseRejectCallback:
2023/11/24 13:34:40 | stdout | Saving image 'inline image' as attachment into note 'LmgibsjtmmzB'
2023/11/24 13:34:40 | stdout | Saving image 'inline image' as attachment into note 'LmgibsjtmmzB'
2023/11/24 13:34:40 | stdout | Saving image 'inline image' as attachment into note 'LmgibsjtmmzB'
2023/11/24 13:34:40 | stdout | Saving image 'inline image' as attachment into note 'LmgibsjtmmzB'
2023/11/24 13:34:40 | stdout | Saving image 'inline image' as attachment into note 'LmgibsjtmmzB'
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | RangeError: Maximum call stack size exceeded
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | ^
2023/11/24 13:34:40 | stderr | }
2023/11/24 13:34:40 | stderr | /usr/src/app/src/services/notes.js:894
2023/11/24 13:34:40 | stderr | Exception in PromiseRejectCallback:
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | RangeError: Maximum call stack size exceeded
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | ^
2023/11/24 13:34:40 | stderr | }
2023/11/24 13:34:40 | stderr | /usr/src/app/src/services/notes.js:894
2023/11/24 13:34:40 | stderr | Exception in PromiseRejectCallback:
2023/11/24 13:34:40 | stderr |  
2023/11/24 13:34:40 | stderr | RangeError: Maximum call stack size exceeded

`

**Comment by Heidieker** [comment]
Okay i found a workaround.
I started an instance on my computer and imported the document.db. After that i made a backup and put it on my server instance without problems.
Before the database had 82 GB now it has 78 GB.

**Reply by Nriver** [reply]
Does the 78 GB file encompass both the document.db and its backups, or is it solely the document.db? The size is quite substantial, sparking my curiosity. I've always been wondering about Trilium's data-handling capacity.

**Reply by Heidieker** [reply]
Sorry, i told crap.
It was MB and not GB.
document.db only

Exactly:
From
82,2 MB (86,196,22 Bytes)
to:
76,5 MB (80,220,160 Bytes)

**Comment by zadam** [comment]
Hello, which version of Trilium you used? A bug looking like this was present in some 0.61.X beta versions.

**Reply by Heidieker** [reply]
My Instance is always pulling the latest stable repisority automatically.
